examples/sushi/models/customer_revenue_by_day.sql

```diff
@@ -2,39 +2,39 @@
 MODEL (
   name sushi.customer_revenue_by_day,
   kind incremental_by_time_range (
-    time_column (ds, 'YYYY-MM-dd'),
+    time_column date,
     batch_size 10,
   ),
   owner jen,
   cron '@daily',
   dialect hive,
   tags expensive,
-  grain [customer_id, ds],
+  grain [customer_id, date],
 );
 
 WITH order_total AS (
   SELECT
     oi.order_id AS order_id,
     SUM(oi.quantity * i.price) AS total,
-    oi.ds AS ds
+    oi.date AS date
   FROM sushi.order_items AS oi
   LEFT JOIN sushi.items AS i
-    ON oi.item_id = i.id AND oi.ds = i.ds
+    ON oi.item_id = i.id AND oi.date = i.date
   WHERE
-    oi.ds BETWEEN '{{ start_ds }}' AND '{{ end_ds }}'
```

Comment:
why did you change this

Reply:
They need to be changed from date strings to dates, and `{{ start_date }}` errored out.

I'll run it again and grab the error.

Reply:
you can make it CAST('{{start_ds}}' as DATE)

Reply:
ok - one issue is that it needed the `JINJA_QUERY_BEGIN`, but with that in place duckdb doesn't render `{{ start_date }}` correctly: 

"`oi`.`event_date` <= 2020 AND `oi`.`event_date` >= 2020"

```
2023-11-29 17:39:24,838 - MainThread - sqlmesh.core.model.definition - DEBUG - Rendered query for 'sushi.customer_revenue_by_day':
  WITH `order_total` AS (
    SELECT
      `oi`.`order_id` AS `order_id`,
      SUM(`oi`.`quantity` * `i`.`price`) AS `total`,
      `oi`.`event_date` AS `event_date`
    FROM `sushi`.`order_items__fixture` AS `oi`
    LEFT JOIN `sushi`.`items__fixture` AS `i`
      ON `oi`.`event_date` = `i`.`event_date` AND `oi`.`item_id` = `i`.`id`
    WHERE
      `oi`.`event_date` <= 2020 AND `oi`.`event_date` >= 2020
    GROUP BY
      `oi`.`order_id`,
      `oi`.`event_date`
  )
  SELECT
    CAST(`o`.`customer_id` AS INT) AS `customer_id`, /* Customer id */
    CAST(SUM(`ot`.`total`) AS DOUBLE) AS `revenue`, /* Revenue from orders made by this customer */
    CAST(`o`.`event_date` AS DATE) AS `event_date` /* Date */
  FROM `sushi`.`orders__fixture` AS `o`
  LEFT JOIN `order_total` AS `ot`
    ON `o`.`event_date` = `ot`.`event_date` AND `o`.`id` = `ot`.`order_id`
  WHERE
    `o`.`event_date` <= 2020 AND `o`.`event_date` >= 2020
  GROUP BY
    `o`.`customer_id`,
    `o`.`event_date` (definition.py:986)
```